README.md

```diff
@@ -275,20 +619,68 @@ We can discuss what is automated, what is not, and how that can be improved. Mor
+> sync up on it. There are more reasons than just RSE-ops to have a common software stack for HPC
+> at this level.  It also helps us push on vendors.
+
 ## Summary of Opportunities
 
 The following areas of RSE-ops are not well developed, and we have opportunity to build or seek out tools to fill the space.
 
 - **Application Development**: Development framework for interactive (browser based) applications for HPC. This relies on being able to ensure secure access in a browser.
+
+> I think people are calling this "cloud services" or "web applications". If you say "app
+> development in HPC people think simulation.
+
 - **Automated Scaling**:  Being able to scale a job, task, or service without thinking about it.
 - **On Demand Resources**: An HPC user should theoretically be able to request a custom resource, and "throw it away" when done. For HPC an administrative team would still need to monitor and maintain the resources, but to the user there could be a more interactive, customized experience. E.g., "I want an instance with this amount of memory, storage, and with this software installed. And I want to customize and request it in a web interface."
+
+> confused about this one because this is what SLURM and Flux do. You can ask for these things, and
+> you are given a temporary allocation. The allocation is then used by someone else when you are
+> done. Allocation is bare metal, not instance-based, but this is what RMs do. The web interface
+> part wuld be new but people are generally managing complex resource requests for ensembles. I
+> think resource requests are more like configuration-as-code than they are like ewb forms -- at
+> least for big apps and studies.
+
 - **Continuous Integration**: Application developers on HPC rarely test their code from version control directly on systems where they are intended to be run. It should be possible to better connect where code lives to where it will ultimately be used to run tests and builds.
+
+> this isn't true -- many times they *only* test it where it is intended to run, and running it
+> elsewhere is the issue when the need arises. It should be easier to do multi-platform CI so that
+> teams could envision supporting more platforms/HPC centers -- e.g. if there was a build/CI farm
+> for HPC.
+
 - **Dependency Management**: With spack, easybuild, and containers, we are doing fairly well. However, constant training and education about these projects is still a challenge.
+
+> also they usability is not what it is for many language ecosystems -- we need reliable binary
+> deployment, which requires much of the metadata you mention above for containers. Distributing
+> binaries and distributing containers require very similar metadata in my mind -- they're related
+> and we probably need both.
+
 - **Community Standards**: Have one or more representatives join the OCI community.
+
+> not just OCI -- other standards bodies as well -- also do we need some foundation of our own for
```

Comment:
Hmm that would be neat, what do you have in mind?

Reply:
well there is CNCF -- I have thought about starting a foundation specifically for lab OSS that people could buy into.  OpenHPC did something like this -- in that they are a Linux Foundation project now, and companies can buy into their project.

The nice thing about this is that it's a way to get money specifically for the project, to pay for small stuff -- overhead, meetings, organizing, governance, slack instances, whatever.  The downside is that IMO the governance structure is overly formal (at least with LF), so it's a lot of process and not a lot gets done. I think with a foundation of our own, or maybe with a bit more thought about governance, we could fix the latter issue.

Reply:
oooh I'm down! This sounds like it would be fun.